docs/code-reviews.md

```diff
@@ -1,4 +1,67 @@
+seeking. For example, are you looking for a check of the implementation of the logic or methods 
+outlined in the protocol (in which case, linking the protocol can aid this), or are you looking for a 
+review of how well you document your code and if it is clear and understandable? Being transparent 
+will help your reviewer do a better job quicker. 
+ 
+## When to ask for Code Review?
+This question's answer will vary from user to user, but it is most useful with every change to the code. In fact, when we surveyed the OpenSAFELY team, most people reported they would like reviews to be as frequent as possible. A good way of conceptualising when to ask for a review is to think about commits and branches. A user makes a branch to change a feature or adding some new analysis, and while making that change, they do several commits to the new branch. When the time comes to merge this back to master, there is a natural inflexion point to do a code review. When you are making a Pull Request, you can tag a reviewer easily. 
+
+Merging a new branch should not be your only trigger for a review. If you are stuck or unsure what direction to take, a code review can clarify what path to take. In general, the more frequent the reviews, the better quality the code tends to be. 
+
+## How can I be a good review-ee?
+Experiences of reviewing were gathered during the code review survey, and some best practices are listed below. 
+
+- Good descriptive names for Pull Requests and commits help the reviewer follow the logic of the code and the changes made (see example of commits below)
+- A clear description of the expectations of the review is helpful, for example, are you expected to run the code or just review it in the browser?
+- Multiple smaller Pull Requests are easier than one large 
+- Code comments and documentations make the logic easier to follow 
+- Naming the order of the files, for example, `01_Data_Extraction.py`, `02_Data_Cleaning.py`, etc. 
+
+#### Examples of good commit messages
+![image](https://user-images.githubusercontent.com/25401512/109499651-166aaf80-7a8d-11eb-86de-f4a0b549fc90.png)
+
+
+## How can I be a good reviewer?
+A survey of the OpenSAFELY team about code reviewers revealed some excellent practices for going forward, and adopting these as a team will improve our overall reviewing procedure and quality. 
+
+- Code review should be a collaborative conversation and don't be afraid to check assumptions or clarify intentions within the review (you can add messages to the end of the review request). 
+- Actionable feedback with suggestions is helpful. 
+- Being clear about what exactly you have reviewed or how thorough the review was—for example, being clear if you read through the code but did not run it (and therefore may miss running errors). 
+- Answer specific questions in the review request.
+- Avoiding jabs on coding style. 
+- Being clear if you do not have time to do a thorough review 
```

Comment:
I like the convention to be clear if something is a nice-to-have-but-not-important - prefixing with `Nit: ` is a good signal for this. e.g. `Nit: I think this would be clearer you didn't prefix with an underscore`

(Sorry I never responded in the survey :guiltyface:)